packages/neuron-wallet/src/services/wallets.ts

```diff
@@ -45,17 +48,49 @@ export class FileKeystoreWallet implements Wallet {
       throw new IsRequired('Name')
     }
 
+    if (!extendedKey && !device) {
```

Comment:
Is this tester supposed to be `!extendedKey || !device`?

Reply:
It requires either `extendedKey` or `device`. If both are missing, it throws an error.